startup/00-nsls2-tools.py

```diff
@@ -1,42 +1,7 @@
-###############################################################################
```

Comment:
Does the fccd need anything of this stuff that was just removed in order to function properly.  I do not think it will ever go back in tardis because that is too much work.  If it were repaired, it could be uses as an SAXS detector.  I am not sure about the chances of that, but I just want to understand the impact of merging this.

Reply:
You can see here that the exact functionality exists in ophyd now so we don't need to override the behavior anymore: https://github.com/bluesky/ophyd/pull/959/files

Reply:
excellent.  good to move the profile into modern times!